pydoclint/main.py

```diff
@@ -209,6 +237,7 @@ def validateStyleValue(
     ),
     is_eager=True,
     callback=injectDefaultOptionsFromUserSpecifiedTomlFilePath,
+    default='pyproject.toml',
```

Comment:
Look at this change:
I think it could be useful for most of users

Reply:
Hey, I'm sorry but I don't quite understand your comment here.

Reply:
I think that now for most of the projects to keep the `pyproject.toml` file in the root is the standard